### ISSUE TITLE
Stop update？

### ISSUE THREAD

**Issue by vincent1912** [Azure/DotNetty#542]
I want to ask whether this project has stopped maintenance because it has not been updated for too long.

**Comment by ScarletKuro** [comment]
This repository is dead. We asked to give this repository to the @dotnet-foundation so that community could maintain it, but @nayato hasn't replied.

**Comment by JayArrowz** [comment]
when

**Comment by iamkisly** [comment]
> This repository is dead. We asked to give this repository to the @dotnet-foundation so that community could maintain it, but @nayato hasn't replied.

Apparently, the project was only a port of the original netty, and will not go beyond its functionality. Why not create a fork in which you can add the required functionality?

**Comment by ScarletKuro** [comment]
> Apparently, the project was only a port of the original netty, and will not go beyond its functionality. Why not create a fork in which you can add the required functionality?

Yes, this is a port of java netty with almost 1:1 its functionality, but it's still missing features like compression, some of codecs etc, and the current implementation has still bugs, not even saying that original netty doesn't stay on place. Yes, you can fork, but you need not only to add the required functionality but maintain it also, which is not a trivial task for such big project as netty.
It's sad that such project stopped to update, because netty is really popular client-server framework and Akka.Net was using it as transport, but as far as i know they will drop dottnety from the project, and many others will.

**Comment by TailyFair** [comment]
@ScarletKuro What Akka.NET switched to from DotNetty?

We use DotNetty in production environment and seeing how unmaintained DotNetty is looks like a risk to keep using it in Production.

**Comment by ScarletKuro** [comment]
@TailyFair custom as far as I understand.

Same, dropped DotNetty as well from my project.

**Comment by fanliang11** [comment]
I don’t agree with your opinion. Dotnetty is not a 1:1 copy function. It adds .NET syntax features. Dotnetty’s core components Channel, EventLoop, ChannelFuture, ChannelHandler, ChannelPipe, FrameDecoder, Libuv have all been implemented, and even extended mqtt, Protocols such as websocket, http, etc., and these functions are already stable and do not need to be greatly modified and upgraded. Besides the basic commonly used codec protocols, other implementations of netty are not perfect. And the ratio of Akka.Net and dotnetty you said is not at the same level. In addition, JAVA rarely uses akka. As a dotnetty community project, you can fork

**Comment by gsw945** [comment]
I found [SpanNetty](https://github.com/cuteant/SpanNetty), it's an active  fork from [DotNetty](https://github.com/Azure/DotNetty)
